hw2/03-count.js

```diff
@@ -1,3 +1,33 @@
 // Add your code here
+const input = document.querySelector('input');
+const textToSearchFrom = document.querySelector('p');
+const originalText = textToSearchFrom.innerHTML;
 
-input.addEventListener('keydown', handleKeyDown);
+input.addEventListener('keydown', handleInput);
+
+function handleInput(event) {
```

Comment:
I know it wasn't specified in the description so take it or leave it, but since the keydown event happens before the value of the input is updated, maybe hold off searching until the user presses the enter key. The results can be kind of wonky with the output being one character behind, especially if the user moves the carrot back and makes a change.

Reply:
Fair point, but I like it the way it is better honestly.